_posts/2016/04/2016-04-03-AMY-release-151.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+authors: ["Piotr Banaszkiewicz"]
+title: "AMY release v1.5.1"
+date: 2016-04-03
+time: "11:00:00"
+category: ["AMY", "Google Summer of Code", "Tooling"]
+---
+
+We're very pleased to announce the newest ([v1.5.1][]) release of [AMY][], our
+workshop management tool.  This release is special because:
+
+1. it contains more bugfixes than new features
+2. we had a number of submissions from prospect
+   [Google Summer of Code 2016](https://summerofcode.withgoogle.com/) students.
```

Comment:
Do you want me to change "prospect [Google Summer of Code 2016](https://summerofcode.withgoogle.com/) students" to "prospect (GSoC) students"?